fernando-avellar/1.iii/index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Exercicio 1.iii</title>
+</head>
+
+<body>
+    <h1>Livro Online</h1>
+
+    <h2>Sumário</h2>
+    <ul>
+        <li><a href="#agradecimentos">Agradecimentos</a></li>
+        <li><a href="#sobreOAutor">Sobre o autor</a></li>
+        <li><a href="#introducao">Introducao</a></li>
+        <li><a href="#capitulo1">Capítulo 1</a></li>
+        <li><a href="#capitulo2">Capítulo 2</a></li>
+        <li><a href="#capitulo3">Capítulo 3</a></li>
+        <li><a href="#apendicea">Apêndice A</a></li>
+        <li><a href="#bibliografia">Referências Bibliográficas</a></li>
+    </ul>
+
+    <a name="agradecimentos"></a>
+    <h2>Agradecimentos</h2>
```

Comment:
Instead of use `<a name="agradecimentos"></a>` try `<h2 id="agradecimentos">Agradecimentos</h2>`